fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -2552,9 +2554,10 @@ private List<Long> chosenBackendIdBySeq(int replicationNum, TStorageMedium stora
         return chosenBackendIds;
     }
 
-    private List<Long> chosenBackendIdBySeq(int replicationNum) throws DdlException {
+    private List<Long> chosenBackendIdBySeq(int replicationNum, Multimap<String, String> locReq) throws DdlException {
         SystemInfoService systemInfoService = GlobalStateMgr.getCurrentSystemInfo();
-        List<Long> chosenBackendIds = systemInfoService.seqChooseBackendIds(replicationNum, true, true);
+        List<Long> chosenBackendIds = systemInfoService.getNodeSelector()
+                .seqChooseBackendIds(replicationNum, true, true, locReq);
```

Comment:
Give more details when there is no enough backend? because disk capacity, or local tag, or medium not match, or alive backend is less than replication_num?

Reply:
For now, when creating table with location property, it's best-effort matching logic, i.e. if there are not enought backends matching the location requirement, we will ignore the location requirement, so the error message can remain the same. And the orignal error do need some refactoring, but for now the `seqChooseBackendIds` api cannot return the detailed error reason from the underlying implementation, to achieve this, we need more refactoring, i would like to put this in another pr, you can assign a jira to me first.